From 741e50a5c7253b720ac127ad969e156e8264a157 Mon Sep 17 00:00:00 2001
From: zhangqing <zhangqing@rock-chips.com>
Date: Sat, 10 Oct 2015 07:39:11 -0700
Subject: [PATCH] rk3228: clk: enable aclk_dmac_bus when clk init

modify enable aclk_dmac_bus control bits.
make aclk_dmac_bus enabled when clk init.

Change-Id: I670dbaabcf9ba6cf9ce683090b49c72e4cbd0410
Signed-off-by: zhangqing <zhangqing@rock-chips.com>
---
 arch/arm/boot/dts/rk3228-fpga.dts | 2 +-
 arch/arm/boot/dts/rk3228.dtsi     | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/rk3228-fpga.dts b/arch/arm/boot/dts/rk3228-fpga.dts
index ccbd7ba8f6ca..a91a869a4a83 100644
--- a/arch/arm/boot/dts/rk3228-fpga.dts
+++ b/arch/arm/boot/dts/rk3228-fpga.dts
@@ -215,7 +215,7 @@
 			<&pclk_bus>,
 			<&clk_gates8 0>,/*aclk_intmem*/
 			<&clk_gates8 1>,/*clk_intmem_mbist*/
-			<&clk_gates8 3>,/*aclk_dmac_bus*/
+			<&clk_gates8 2>,/*aclk_dmac_bus*/
 			<&clk_gates10 1>,/*g_aclk_bus*/
 			<&clk_gates13 9>,/*aclk_gic400*/
 			<&clk_gates8 3>,/*hclk_rom*/
diff --git a/arch/arm/boot/dts/rk3228.dtsi b/arch/arm/boot/dts/rk3228.dtsi
index 444ac875f2bf..1d46bcb005ad 100644
--- a/arch/arm/boot/dts/rk3228.dtsi
+++ b/arch/arm/boot/dts/rk3228.dtsi
@@ -132,7 +132,7 @@
 			<&pclk_bus>,
 			<&clk_gates8 0>,/*aclk_intmem*/
 			<&clk_gates8 1>,/*clk_intmem_mbist*/
-			<&clk_gates8 3>,/*aclk_dmac_bus*/
+			<&clk_gates8 2>,/*aclk_dmac_bus*/
 			<&clk_gates10 1>,/*g_aclk_bus*/
 			<&clk_gates13 9>,/*aclk_gic400*/
 			<&clk_gates8 3>,/*hclk_rom*/
-- 
2.35.3

